src/proof_system/prover.rs

```diff
@@ -145,6 +147,30 @@ impl Prover {
         self.preprocessed_transcript.append_message(label, message);
     }
 
+    /// Adds the blinding scalars to a given vector. Always the same elements
+    /// of 'w_vec' are modified at the beginning of it, and appended at the end:
+    /// if hiding degree = 1: (b2*X(n+1) + b1*X^n - b2*X - b1) + w_vec
+    /// if hiding degree = 2: (b3*X^(n+2) + b2*X(n+1) + b1*X^n - b3*X^2 - b2*X
+    /// - b1) + w_vec
+    pub(crate) fn blind_poly<R: RngCore + CryptoRng>(
+        w_vec: &Vec<dusk_bls12_381::BlsScalar>,
```

Comment:
Above `dusk_bls12_381::BlsScalar` is already imported, so you can simply use `&Vec<BlsScalar>` here.

Reply:
thanks! :)